src/devices/tech.ts

```diff
@@ -0,0 +1,103 @@
+import {ota} from '../index';
```

Comment:
Please also add this file to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/index.ts